### ISSUE TITLE
Problem with build on Windows 10 VS 2015

### ISSUE THREAD

**Issue by dmitriyparasochka** [CryptoManiac/AssemblyLoader#50]
I've tried build project with Platform toolset:  "Visual Studio 2015 (v140)"

```
Error	C2228	left of '.guid' must have class/struct/union	PicoVM	e:\oss\assemblyloader-master\assemblyloader-master\picovm\clr\assemblydata.cxx	445	
Error	C2039	'module': is not a member of 'AssemblyData::CLIMetaDataTables'	PicoVM	e:\oss\assemblyloader-master\assemblyloader-master\picovm\clr\assemblydata.cxx	445	
Error (active)		class "AssemblyData::CLIMetaDataTables" has no member "module"	PicoVM	e:\OSS\AssemblyLoader-master\AssemblyLoader-master\PicoVM\CLR\AssemblyData.cxx	445	
```

Do you have any ideas how fix this errors?

PS. I'm not C++ developer.

**Comment by dmitriyparasochka** [comment]
Maybe this string need change https://github.com/CryptoManiac/AssemblyLoader/blob/master/PicoVM/CLR/AssemblyData.cxx#L445 to
`    return cliMetaDataTables._Module[0].guid;` ?

This string fixed my problem with build project.

**Comment by CryptoManiac** [comment]
Yep, that solution is absolutely correct. Fixed in 16aa0e571f1c1154535c00edda482ccbf02674c4.
